src/templates/jira.html

```diff
@@ -0,0 +1,216 @@
+                   
+                        
+                   
+                    
+                </div>
+                </div>
+
+
+            </div>
+
+            <div class = 'col-sm-1 close' >
+                <i class="far fa-window-close"></i>
+            </div>
+        </div>
+    </div>
+       
+    <div class = 'information hide'>
+          
+                <div class = 'title'>
+                    Here goes the title
+                </div>
+
+                <div class = 'container'></div>
+                <div class='row'>
+                    <div class = 'col-sm-6 left '>
+                        Estimate:
+                    </div>
+                    <div class='col-sm-6 estimate'>
+                        Unestimated
+                    </div>
+                </div>
+                <div class='row'>
+                    <div class = 'col-sm-6 left'>
+                        Remaining:
+                    </div>
+                    <div class='col-sm-6'>
+                        <input type="text"  id='remaining2'>
+                    </div>
+                </div>
+              
+                
+                <div class='det left'>
+                    <i class="fas fa-chevron-circle-down"></i>
+                    Details 
+                </div>
+               
+
+                <div class = 'add hide'>
+                    <div class='row'>
+                        <div class = 'col-sm-6 left'>
+                            Status:
+                        </div>
+                        <div class='col-sm-6 st'>
+                            CLOSED
+                        </div>
+                    </div>
+
+                    <div class='row'>
+                        <div class = 'col-sm-6 left'>
+                            Labels:
+                        </div>
+                        <div class='col-sm-6'>
+                            None
+                        </div>
+                    </div>
+                </div>
+                <button><i class="fas fa-trash-alt"></i></button>
+            </div>
+       
+            
+    </div>
+        
+    <div class = 'changeColName hide' id='changeColName'>
+
+     
+            <div class="row">
+                <div class='col-sm '>
+                    New column name:
+                        <span class = 'error2 error'>Must contain at least 3 charachters</span>
+                        <input type="text" class='colName'>
+                        <button class='saveCol'>Save</button>
+                </div>
+                <div class='col-sm-1 close'>
+                    <i class="far fa-window-close"></i>
+                </div>
+            </div>
+        
+        
+    </div>
+
+    <div class = 't'>
+    <h2>Drag and Drop</h2>
+    <p>Drag the card back and forth between columns.</p>
+    </div>
+
+    <script>
+        $(document).ready(function(){
+          $('[data-toggle="popover"]').popover({ trigger: "hover" });
```

Comment:
This is the only place jquery is used. Can you please remove it and use simple js ?